api/spec/factories/session.rb

```diff
@@ -5,25 +5,32 @@
   sequence(:year, Time.now.year)
```

Comment:
If we generate a session with the summer trait, the name would be "Summer #{current_year}", but if we then want to generate a fall session in the same year, would the sequence force the year to be current_year + 1?

Reply:
Using the sequence, yes it will generate to curr_year + 1.

However, we can override the attributes in specs.